docs/components/attention/react.md

```diff
@@ -32,16 +32,29 @@ function Example() {
         ref={targetEl}
         onMouseEnter={() => setShow(true)}
         onMouseLeave={() => setShow(false)}
+        onMouseEnter={() => setShow(true)}
+        onMouseLeave={() => setShow(false)}
+        onFocus={() => setShow(true)}
+        onBlur={() => setShow(false)}
+        onKeyDown={handleKeyDown}
+        tabIndex={0}
       >
-        hover this for useless info
+        <button
+          aria-describedby='tooltip-bubbletext'
```

Comment:
Should it be `aria-details` or is `aria-describedby` more correct here?

Reply:
The a11y-team had `aria-describedby` in the example in the accesibility-audit-warp-document. :)

Reply:
I see, they described their example with "This code will announce the tooltip text regardless of visibility", so that makes sense in the case of Tooltip, which probably is mostly opened on hover.